src/ResourceManager/Network/Commands.Network/NetworkWatcher/SetAzureNetworkWatcherConfigFlowLogCommand.cs

```diff
@@ -153,6 +153,22 @@ public class SetAzureNetworkWatcherConfigFlowLogCommand : NetworkWatcherBaseCmdl
         [ValidateRange(1, int.MaxValue)]
         public int RetentionInDays { get; set; }
 
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Type of flow log format.")]
+        [ValidateNotNullOrEmpty]
+        [PSArgumentCompleter("Json")]
+        public string FormatType { get; set; }
+
+        [Parameter(
+            Mandatory = false,
+            ValueFromPipelineByPropertyName = true,
+            HelpMessage = "Version of flow log format.")]
+        [ValidateNotNull]
+        [ValidateRange(1, int.MaxValue)]
+        public int? FormatVersion { get; set; }
```

Comment:
@zwswim nit: `int?` --> `int` (since you have the `ValidateNotNull` attribute)

Reply:
Will this affect existing user ?  If they donot set the TrafficAnalyticsInterval ?

Reply:
It depends on what you want the default value to be if the user doesn't provide a value for `-FormatVersion`, if you want it to be 0, then you should just change the value to `int`, if you want it to be `null`, then you should just remove `ValidateNotNull`.

Reply:
@cormacpayne   fix by remove the "ValidateNotNull"